packages/react-sdk/src/generator.test.ts

```diff
@@ -10,13 +10,24 @@ const createPage = (path: string) => ({
   meta: {},
 });
 
+const pages = {
+  rootFolder: {
+    id: "root",
+    name: "",
+    slug: "",
+    children: [],
+  },
+  meta: {},
+  pages: [],
+  folders: [],
```

Comment:
Why not treat the first folder as the root one?

Reply:
discussed, will use an id as a root pointer